utils/plottools.py

```diff
@@ -15,10 +15,7 @@ def __init__(self, exp, figpath, dims, units, ticks=None):
     def _default_dim_ticks(self, nticks=11):
```

Comment:
nticks need to be removed, it can be change in the future

Reply:
I removed this parameter in branch "set_ticks_by_datetime_variable" pull request